requirements/verification-by-reference.md

```diff
@@ -343,7 +343,7 @@ The reverse lookup index could be expanded as follows:
 
 ![Multi-platform manifest with SBoM, TUF, and Scan artifacts](./media/oci-manifest-index-reverse-lookup-sbom.png)
 
-The user still references `registry.contoso.com/web:a2b2`, while other parts of the system that adhere to notary v2 would know to query the registry for the additional verification information.
+The user still references `registry.contoso.com/web:a2b2`, while other parts of the system that adhere to notary Project would know to query the registry for the additional verification information.
```

Comment:
The user still references `registry.contoso.com/web:a2b2`, while other parts of the system that adhere to **the Notary** Project